apps/glusterfs/dbcommon.go

```diff
@@ -69,3 +71,87 @@ func initializeBuckets(tx *bolt.Tx) error {
 
 	return nil
 }
+
+func DeleteBricksWithEmptyPath(db *bolt.DB, all bool, clusterIDs []string, nodeIDs []string, deviceIDs []string, debug bool) error {
```

Comment:
I'm not in love with this function signature. I won't object to it in the short term but I would have gone for an approach where the all flag populates a simple struct with the ID list and the id level (eg. clear all bricks matching id X, vs. clear all bricks under cluster Y, and made them mutually exclusive. That said, this seems OK for now.

Reply:
Agreed. Needs to be refactored.